### PR TITLE
docs(stats/base/dists/arcsine): improve examples

### LINKED ISSUE #1612: [RFC]: improve README examples of `stats/base/dists/arcsine` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/arcsine#examples) of the `stats/base/dists/arcsine` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/arcsine) of the `stats/base/dists/arcsine` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `stats/base/dists/arcsine` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by mayank1365**
Resolves #1612 

## Description

> What is the purpose of this pull request?

This pull request:

- a. Enhances Documentation: Improves the README examples for the stats/base/dists/arcsine namespace. It adds more detailed and varied examples to better illustrate the use of different functions available in the arcsine distribution, such as cdf, pdf, mean, variance, and others.

- b. Provides Practical Usage: Includes practical examples demonstrating the calculation of statistical properties (e.g.,  median, mode, entropy) and the generation of random numbers. This helps users understand how to apply the library functions in real-world scenarios and ensures they can effectively utilize the arcsine distribution in their own projects.

## Related Issues

> Does this pull request have any related issues?

- resolves: Improves documentation clarity by adding more comprehensive examples for the stats/base/dists/arcsine namespace.
- fixes: Updates and corrects existing examples to better demonstrate the usage of various functions and properties related to the arcsine distribution.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by mayank1365** [comment]
@Planeshifter please have a look at the improved examples for ```stats/base/dists/arcsine```

**Review by Planeshifter — COMMENTED** [review]
Can you please move everything inside a single example code block? 

You should also update the `examples/index.js` file to match the example code block from the `README.md`.

**Comment by mayank1365** [comment]
> Can you please move everything inside a single example code block?
> 
> You should also update the `examples/index.js` file to match the example code block from the `README.md`.

@Planeshifter I have done the required changes to ```examples/index.js``` and I have moved everything inside a single example code block in the ```README.md```.

**Comment by mayank1365** [comment]
@kgryte @Planeshifter  please merge this PR, you seem to have closed the issue before the merger of the PR.

File: examples/index.js
Comment (kgryte) [inline examples/index.js:1]: This file should not have been modified.

Comment (kgryte) [reply examples/index.js:1]: Please revert these changes.

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md [hunk 1]
```diff
@@ -107,10 +107,36 @@ var mu = dist.mean;
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+// Display Available Functions
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md:110]: Remove this comment.

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md [hunk 2]
```diff
@@ -107,10 +107,36 @@ var mu = dist.mean;
...
 var objectKeys = require( '@stdlib/utils/keys' );
 var arcsine = require( '@stdlib/stats/base/dists/arcsine' );
-
 console.log( objectKeys( arcsine ) );
+
+// Calculate the Median for the arcsine distribution in the range [0, 1]
+var medianValue = arcsine.median(0, 1);
+console.log('The median of the distribution is', medianValue);
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md:117]: Follow spacing conventions as followed in the project.

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md [hunk 3]
```diff
@@ -107,10 +107,36 @@ var mu = dist.mean;
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+// Display Available Functions
 var objectKeys = require( '@stdlib/utils/keys' );
 var arcsine = require( '@stdlib/stats/base/dists/arcsine' );
-
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md:112]: Why did you remove this line?

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md [hunk 4]
```diff
@@ -107,10 +107,36 @@ var mu = dist.mean;
...
+// Calculate the Median for the arcsine distribution in the range [0, 1]
+var medianValue = arcsine.median(0, 1);
+console.log('The median of the distribution is', medianValue);
+
+// Calculate the Mode for the arcsine distribution in the range [0, 1]
+var modeValue = arcsine.mode(0, 1);
+console.log('The mode of the distribution is', modeValue);
+
+// Calculate the Entropy for the arcsine distribution in the range [0, 1]
+var log = require( '@stdlib/math/base/special/log' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md:124]: Why is this here?

**Review by kgryte — CHANGES_REQUESTED** [review]
This PR needs to be updated before it can be considered for review.

File: examples/index.js
Comment (mayank1365) [reply examples/index.js:1]: Planetshifter told to change that file as well.

Comment (kgryte) [reply examples/index.js:1]: No, he meant the `examples/index.js` in the namespace package, not in the root directory.

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md (hunk 4 above)
Comment (mayank1365) [reply lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md:124]: This was a requirement in the issue description

> [Examples may require functionality from other packages to help make examples more interesting.](https://github.com/stdlib-js/stdlib/issues/1612)

File: examples/index.js
Comment (mayank1365) [reply examples/index.js:1]: oh ok I will fix the files accordingly and revert the necessary files back

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md (hunk 4 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md:124]: The import must make sense. As is, you don't actually use `log` anywhere in the examples.

File: examples/index.js
Comment (mayank1365) [reply examples/index.js:1]: quick doubt how should i handle this, i am getting this error in ```examples/index.js``` in root as well as the namespace package
```
  25:1  warning  Unexpected console statement                                     no-console
  28:1  error    All 'var' declarations must be at the top of the function scope  vars-on-top
  29:1  warning  Unexpected console statement                                     no-console
```

File: lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md (hunk 4 above)
Comment (mayank1365) [reply lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md:124]: understood, i have fixed it

File: examples/index.js
Comment (mayank1365) [reply examples/index.js:1]: @kgryte
